nb.rs,"HTTPSConnectionPool(host='nb.rs', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.nb.rs', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:1124)')))"
ncc.gov.ng,"HTTPSConnectionPool(host='ncc.gov.ng', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='ncc.gov.ng', port=443): Read timed out. (read timeout=2)"
neik.vn,"HTTPSConnectionPool(host='neik.vn', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='neik.vn', port=443): Read timed out. (read timeout=2)"
neliti.com,"HTTPSConnectionPool(host='neliti.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.neliti.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
nemoshideaway.com,"HTTPSConnectionPool(host='nemoshideaway.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='nemoshideaway.com', port=80): Read timed out. (read timeout=2)"
neopod.ru,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","HTTPConnectionPool(host='neopod.ru', port=80): Read timed out. (read timeout=2)"
neowing.co.jp,"HTTPSConnectionPool(host='neowing.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f73bcc41310>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='neowing.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f73ba44e0d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nestia.com,"HTTPSConnectionPool(host='www.nestia.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.nestia.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
net.ee,"HTTPSConnectionPool(host='net.ee', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f73b9584190>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='net.ee', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f73b9fa8e50>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
netky.sk,"HTTPSConnectionPool(host='netky.sk', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f73bd724c40>, 'Connection to netky.sk timed out. (connect timeout=2)'))","HTTPConnectionPool(host='www.netky.sk', port=80): Read timed out. (read timeout=2)"
neturuguay.com,"HTTPSConnectionPool(host='neturuguay.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='neturuguay.com', port=443): Read timed out. (read timeout=2)"
netwerkgalerij.be,"HTTPSConnectionPool(host='netwerkgalerij.be', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f73b7c0f040>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='netwerkgalerij.be', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f73bce67070>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
newellwind.com,"HTTPSConnectionPool(host='newellwind.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f73bcc414f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='newellwind.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f73bc9a01f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
newpages.com.my,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
news-italia-mondo.it,"HTTPSConnectionPool(host='news-italia-mondo.it', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='news-italia-mondo.it', port=80): Read timed out. (read timeout=2)"
news-mail.com.au,"HTTPSConnectionPool(host='www.couriermail.com.au', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.couriermail.com.au', port=443): Read timed out. (read timeout=2)"
news.com.au,"HTTPSConnectionPool(host='www.news.com.au', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.news.com.au', port=443): Read timed out. (read timeout=2)"
newsformy.com,"HTTPSConnectionPool(host='newsformy.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='newsformy.com', port=443): Read timed out. (read timeout=2)"
newswire.ca,"HTTPSConnectionPool(host='newswire.ca', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='newswire.ca', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
newtv.co.th,"HTTPSConnectionPool(host='newtv.co.th', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='newtv.co.th', port=80): Read timed out. (read timeout=2)"
nguoiduatin.vn,"HTTPSConnectionPool(host='nguoiduatin.vn', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f73b6ccde20>, 'Connection to nguoiduatin.vn timed out. (connect timeout=2)'))","HTTPConnectionPool(host='nguoiduatin.vn', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f73b7086490>, 'Connection to nguoiduatin.vn timed out. (connect timeout=2)'))"
nhattao.com,"HTTPSConnectionPool(host='nhattao.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='nhattao.com', port=443): Read timed out. (read timeout=2)"
nhk.or.jp,"HTTPSConnectionPool(host='nhk.or.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f73b6ae3e80>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nhk.or.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f73b5bd4a30>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ni.rs,"HTTPSConnectionPool(host='ni.rs', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f73b866c970>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ni.rs', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f73bc26ccd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nichepoliambulatorio.it,"HTTPSConnectionPool(host='nichepoliambulatorio.it', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='nichepoliambulatorio.it', port=80): Read timed out. (read timeout=2)"
nihon-kankou.or.jp,"HTTPSConnectionPool(host='nihon-kankou.or.jp', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.nihon-kankou.or.jp', port=443): Read timed out. (read timeout=2)"
nii.ac.jp,"HTTPSConnectionPool(host='nii.ac.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f73bc91c940>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nii.ac.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f73be600f40>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
niigata.lg.jp,"HTTPSConnectionPool(host='niigata.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f73bb4e4760>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='niigata.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f73b577d190>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nikkei.co.jp,"HTTPSConnectionPool(host='nikkei.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f73b65989d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nikkei.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f73b4799bb0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nilephotos.net,"HTTPSConnectionPool(host='nilephotos.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='nilephotos.net', port=443): Read timed out. (read timeout=2)"
ninhbinh.gov.vn,"HTTPSConnectionPool(host='ninhbinh.gov.vn', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='ninhbinh.gov.vn', port=443): Read timed out. (read timeout=2)"
ninhthuan.gov.vn,"HTTPConnectionPool(host='www.ninhthuan.gov.vn', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.ninhthuan.gov.vn', port=80): Read timed out. (read timeout=2)"
nippon.com,"HTTPSConnectionPool(host='nippon.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f73b469c6a0>, 'Connection to nippon.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='nippon.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f73b805cc10>, 'Connection to nippon.com timed out. (connect timeout=2)'))"
njuzz.ee,"HTTPSConnectionPool(host='njuzz.ee', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f73b5ba4880>, 'Connection to njuzz.ee timed out. (connect timeout=2)'))","HTTPConnectionPool(host='njuzz.ee', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f73b4704040>: Failed to establish a new connection: [Errno 113] No route to host'))"
nocowboys.co.nz,"HTTPSConnectionPool(host='nocowboys.co.nz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='nocowboys.co.nz', port=443): Read timed out. (read timeout=2)"
nojima.co.jp,"HTTPSConnectionPool(host='nojima.co.jp', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f73bd573a00>, 'Connection to nojima.co.jp timed out. (connect timeout=2)'))","HTTPConnectionPool(host='nojima.co.jp', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f73b63112b0>, 'Connection to nojima.co.jp timed out. (connect timeout=2)'))"
nomos-elibrary.de,"HTTPSConnectionPool(host='www.nomos-elibrary.de', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.nomos-elibrary.de', port=443): Read timed out. (read timeout=2)"
nonstopshop.rs,"HTTPSConnectionPool(host='nonstopshop.rs', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='nonstopshop.rs', port=443): Read timed out. (read timeout=2)"
novilist.hr,"HTTPSConnectionPool(host='www.novilist.hr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.novilist.hr', port=443): Read timed out. (read timeout=2)"
novine.hr,"HTTPSConnectionPool(host='novine.hr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='novine.hr', port=443): Read timed out. (read timeout=2)"
novosti.hr,"HTTPSConnectionPool(host='novosti.hr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='novosti.hr', port=443): Read timed out. (read timeout=2)"
npru.ac.th,"HTTPSConnectionPool(host='npru.ac.th', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.npru.ac.th', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
npz.sk,"HTTPSConnectionPool(host='npz.sk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1124)')))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
nrsr.sk,"HTTPSConnectionPool(host='nrsr.sk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f73b3d1ea30>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nrsr.sk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f73b9858af0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ns.ac.rs,"HTTPSConnectionPool(host='ns.ac.rs', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f73b7917700>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ns.ac.rs', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f73b69a2220>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nsc.ru,"HTTPSConnectionPool(host='nsc.ru', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f73bc7e4fd0>, 'Connection to nsc.ru timed out. (connect timeout=2)'))","HTTPConnectionPool(host='nsc.ru', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f73b83bc3a0>, 'Connection to nsc.ru timed out. (connect timeout=2)'))"
nsk.hr,"HTTPSConnectionPool(host='nsk.hr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='nsk.hr', port=443): Read timed out. (read timeout=2)"
nso.go.th,"HTTPSConnectionPool(host='nso.go.th', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f73b69a2580>, 'Connection to nso.go.th timed out. (connect timeout=2)'))","HTTPConnectionPool(host='nso.go.th', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f73be4c2a90>, 'Connection to nso.go.th timed out. (connect timeout=2)'))"
nstru.ac.th,"HTTPSConnectionPool(host='nstru.ac.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f73bd67a250>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='nstru.ac.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f73b409ad60>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
nta.go.jp,"HTTPSConnectionPool(host='nta.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f73b3e21a60>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nta.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f73b5b98d30>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nttdocomo.co.jp,"HTTPSConnectionPool(host='nttdocomo.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f73bd7a3790>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nttdocomo.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f73bcbb8ac0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ntua.gr,"HTTPSConnectionPool(host='ntua.gr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f73bc5582e0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='ntua.gr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f73b7c263d0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
nu6i-bg-net.com,"HTTPSConnectionPool(host='nu6i-bg-net.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='nu6i-bg-net.com', port=443): Read timed out. (read timeout=2)"
number1.co.il,"HTTPSConnectionPool(host='number1.co.il', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='number1.co.il', port=80): Read timed out. (read timeout=2)"
nusalist.com,"HTTPSConnectionPool(host='nearbymalaysia.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='nearbymalaysia.com', port=443): Read timed out. (read timeout=2)"
nuv.cz,"HTTPSConnectionPool(host='nuv.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:1124)')))","HTTPConnectionPool(host='www.nuv.cz', port=80): Read timed out. (read timeout=2)"
nwu.ac.za,"HTTPSConnectionPool(host='nwu.ac.za', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f73b5bf69d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nwu.ac.za', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f73ba44e760>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nyf.hu,"HTTPSConnectionPool(host='nyf.hu', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f73bbe35a00>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nyf.hu', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f73b411cc70>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nyitvatartas24.hu,"HTTPSConnectionPool(host='nyitvatartas24.hu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.nyitvatartas24.hu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
oae.go.th,"HTTPSConnectionPool(host='oae.go.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f73bbdad910>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='oae.go.th', port=80): Read timed out. (read timeout=2)"
oaed.gr,"HTTPSConnectionPool(host='oaed.gr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='oaed.gr', port=443): Read timed out. (read timeout=2)"
oamk.fi,"HTTPSConnectionPool(host='www.oamk.fi', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.oamk.fi', port=443): Read timed out. (read timeout=2)"
oblizniprste.si,"HTTPSConnectionPool(host='oblizniprste.si', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='oblizniprste.si', port=443): Read timed out. (read timeout=2)"
obraspublicas.gob.ec,"HTTPSConnectionPool(host='obraspublicas.gob.ec', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'obraspublicas.gob.ec' doesn't match either of 'www.agricultura.gob.ec', 'agricultura.gob.ec'"")))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
obz.hr,"HTTPSConnectionPool(host='obz.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f73b658fd30>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='obz.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f73bc4aed30>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
oca.com.uy,"HTTPSConnectionPool(host='oca.com.uy', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f73bd7f5bb0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='oca.com.uy', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f73b3c28070>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
